rbi/core/unbound_method.rbi

```diff
@@ -1,15 +1,19 @@
 # typed: __STDLIB_INTERNAL
 
 # Ruby supports two forms of objectified methods.
-# [`Class`](https://docs.ruby-lang.org/en/2.6.0/Class.html) `Method` is used to
+# [`Class`](https://docs.ruby-lang.org/en/2.6.0/Class.html)
+# [`Method`](https://docs.ruby-lang.org/en/2.6.0/Method.html) is used to
```

Comment:
Should `Class` here even be linked?  Seems like a bit of noise.  (Or should there even be "Class" starting the sentence?  Compare the `UnboundMethod` sentence below, which has no such introduction.)

Reply:
This is autogenerated from the official documentation.

Reply:
https://docs.ruby-lang.org/en/2.7.0/UnboundMethod.html

We have a script that scrapes this documentation, converts it to markdown style, and inserts the comments in the right places:

```
rbi/tools/sync-rdoc.rb rbi/core/unbound_method.rbi
```